### PR TITLE
add code of conduct and contributing guide

### PR DESCRIPTION
**Pull request by wang-boyu**
Partly addresses https://github.com/projectmesa/mesa-geo/issues/46

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
+Contributing
```
Comment (rht) [inline CONTRIBUTING.rst:1]: Can this file be CONTRIBUTING.md? Having a mix of .md and .rst is surely confusing. I'm aware that this is still the case in projectmesa/mesa.

Comment (wang-boyu) [reply CONTRIBUTING.rst]: Sure, changed it from `.rst` to `.md`.

File: .github/ISSUE_TEMPLATE/asking-help.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Asking for help
+about: If you need help using Mesa, you should post in https://github.com/projectmesa/mesa-geo/discussions
```
Comment (rht) [inline .github/ISSUE_TEMPLATE/asking-help.md:3]: Should be Mesa-Geo?

Comment (wang-boyu) [reply .github/ISSUE_TEMPLATE/asking-help.md]: Fixed!

**Comment by rht** [comment]
LGTM

**Pull request merged**